system-addon/content-src/components/TopSites/_TopSites.scss

```diff
@@ -326,13 +321,54 @@
 }
 
 .topsite-form {
+  .form-input-container {
+    display: flex;
+    max-width: 492px;
+    margin: 0 auto;
+    padding: 32px;
+
+    .top-site-outer {
+      width: 96px;
+      padding: 0;
+      margin: 64px 0 0;
+      margin-inline-start: 32px;
+      list-style: none;
+
+      a {
+        cursor: default;
+      }
+    }
+  }
+
+  label {
+    font-size: 13px;
+  }
+
   .form-wrapper {
-    margin: auto;
-    max-width: 350px;
-    padding: 15px 0;
+    width: 300px;
 
     .field {
       position: relative;
+
+      .clear-input-value {
+        position: absolute;
+        top: 16px;
+        width: 24px;
+        height: 16px;
+        background: $white url('#{$image-path}searchfield-cancel.svg');
```

Comment:
Does this explicit background color need to be set? Might cause oddness if the input gets a different color somehow. If not, probably just set `background-image` ?

Reply:
There is actually no design for this. I was trying to avoid input overflow scenarios where it looks odd with no background color. I will ask for design feedback.
<img width="69" alt="screen shot 2018-02-13 at 09 51 49183" src="https://user-images.githubusercontent.com/810040/36141016-9cc2c718-10a3-11e8-89d3-f068f0f91991.png">
<img width="60" alt="screen shot 2018-02-13 at 09 52 04107" src="https://user-images.githubusercontent.com/810040/36141017-9cdaf5d6-10a3-11e8-9b2d-5c86948244fb.png">

Reply:
> Yep. Some padding is preferred